flax/experimental/nnx/nnx/rnglib.py

```diff
@@ -130,24 +130,25 @@ def __init__(
     }
     self._trace_state = tracers.TraceState()
 
-  def _make_rng(self, name: str) -> jax.Array:
+  def _make_rng(self, name: str, error_type: Exception) -> jax.Array:
```

Comment:
I think presubmits failing because [this test](https://github.com/google/flax/blob/1721f4b31c07a80c73e1440175c2e8adfc16cb15/flax/experimental/nnx/tests/test_rngs.py#L40) is expecting a `ValueError` to be thrown instead of an `Exception`